app-modules/ai/src/Filament/Pages/ManageAiSettings.php

```diff
@@ -145,7 +145,7 @@ public function form(Form $form): Form
                     ->searchable()
                     ->required(),
             ])
-            ->disabled(! auth()->user()->can('product_admin.*.update'));
+            ->disabled(! auth()->user()->can('saas_global_admin.*.update'));
```

Comment:
This should be checking if they are a global admin rather than a permission. There is a helper for this. `$user->isSuperAdmin()`